RocketChat/Rocket.Chat
# fix: engagement dashboard not working

## Proposed changes (including videos or screenshots)
The aggregate function was not being call because of the name in the cron job is being the same of the finish sessions function, resulting in not having information to display on the Engagement Dashboard.





## Issue(s)


## Steps to test or reproduce
1. Use the platform to generate session records;
2. Wait for the cron job to run _(02:00 am)_;
3. Check if any data was generated in `admin > engagement`;


## Further comments

[WM-158](https://rocketchat.atlassian.net/browse/WM-158)


[WM-158]: https://rocketchat.atlassian.net/browse/WM-158?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
